shared/go/aws/cloudwatchlogs/cloudwatch.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright (C) 2022 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudwatchlogs
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk"
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk/plugins/source"
+	"github.com/falcosecurity/plugins/shared/go/aws/session"
+)
+
+// Client represents a client for CloudwatchLogs API
+type Client struct {
+	*cloudwatchlogs.CloudWatchLogs
+}
+
+// CreateClient returns a Client for CloudwatchLogs API
+func CreateClient(cfgs *aws.Config) *Client {
+	return &Client{
+		CloudWatchLogs: cloudwatchlogs.New(session.CreateSession(), cfgs),
+	}
+}
+
+// CreateFilter returns a filter for the query to CloudwatchLogs API
+func CreateFilter(startTime, endTime int64, filterPattern, logGroupName, logStreamNamePrefix, logStreamNames string) *cloudwatchlogs.FilterLogEventsInput {
```

Comment:
startTime and endTime seem to not be used here. Plus, I would make them of type `time.Time`. More broadly, I don't see much value in having this function, which just passes-in its parameters to `cloudwatchlogs.FilterLogEventsInput`.